src/datasets/iterable_dataset.py

```diff
@@ -2074,7 +2075,10 @@ def from_generator(
                 Keyword arguments to be passed to the `generator` callable.
                 You can define a sharded iterable dataset by passing the list of shards in `gen_kwargs`.
                 This can be used to improve shuffling and when iterating over the dataset with multiple workers.
+            split ([`NamedSplit`], defaults to `Split.TRAIN`)
```

Comment:
My bad, I forgot the colon:
```suggestion
            split ([`NamedSplit`], defaults to `Split.TRAIN`):
```